setup.py

```diff
@@ -36,6 +36,7 @@
 
 scripts = ['shell/activate',
            'shell/deactivate',
+           'shell/conda.fish' 
```

Comment:
By the way, would it be possible to patch `config.fish` the same way the installer offers to patch `.bashrc`? Or, at least, give instructions during the installation process.